uk.co.compucorp.civicrm.hrleaveandabsences/tests/phpunit/CRM/HRLeaveAndAbsences/BAO/LeaveRequestTest.php

```diff
@@ -216,6 +216,56 @@ public function testShouldReturnNullIfItCantFindAPublicHolidayLeaveRequestForACo
     $this->assertNull(LeaveRequest::findPublicHolidayLeaveRequest(3, $publicHoliday));
   }
 
+  public function testCanFindAPublicHolidayLeaveRequestReturnsResultWhenNonDeletedIsFalseAndPublicHolidayRequestIsDeleted() {
+    AbsencePeriodFabricator::fabricate([
```

Comment:
You're mixing two name styles here and now, if you try to read the method name, it doesn't make sense. Separating the methods will probably help with the test names